target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -110,17 +110,31 @@ define Device/ubnt-xw
   UBNT_VERSION := 6.0.4
 endef
 
-define Device/ubnt_acb-isp
+define Device/ubnt-acb
   $(Device/ubnt)
-  SOC := qca9533
-  DEVICE_MODEL := airCube ISP
   IMAGE_SIZE := 15744k
-  UBNT_BOARD := ACB-ISP
-  UBNT_CHIP := qca9533
+  UBNT_BOARD := ACB
   UBNT_TYPE := ACB
   UBNT_VERSION := 2.5.0
 endef
-TARGET_DEVICES += ubnt_acb-isp
+
+define Device/ubnt_aircube-isp
+  $(Device/ubnt-acb)
+  SOC := qca9533
+  DEVICE_MODEL := airCube ISP
+  UBNT_CHIP := qca9533
+  SUPPORTED_DEVICES += ubnt,acb-isp
+endef
+TARGET_DEVICES += ubnt_aircube-isp
+
+define Device/ubnt_aircube-ac
```

Comment:
ac before isp

Reply:
thanks,
as we keep changes for acb-isp -> aircube-isp change separately - order would be correct for now.
ubnt,acb-isp comes first and then ubnt,aircube-ac

i will swap them around making aircube-ac first with mr that renames acb-isp -> aircube-isp.